src/components/tables/AppsDataTable.vue

```diff
@@ -78,7 +78,7 @@ export default {
       this.apps.forEach(app => {
         this.rows.push(
           [
-            { value: app.name },
+            { value: { title: app.name, appId: 272316 }, type: 'action' },
```

Comment:
what's this id?

Reply:
@squallstar this is an example of the app ID taken from my staging account for this app.
```
{
  "apiHost": "https://staging.api.fliplet.com",
  "userId": 37875,
  "email": "yaroslav.o@upplabs.com",
  "auth_token": "eu--session--c30dcbcc307a7d1f3370fce91265c166-779-059-5046",
  "appId": 272316,
  "appName": "Charts",
  "pageId": 732121,
  "pageTitle": "Charts"
}
```
To be able to show us how the app overlay will work.
When API will be ready I will change it to the value from the API because for now we don't have an example of how it will be looking In the API [example ](https://docs.google.com/document/d/1eKD_dnz7n8VPJIQAPqSzOaEU74Ct63GZmxFi7t-bBvE/edit?ts=5eb974d2#heading=h.8fljb4jw2vy) you have give us.
But if you can give me an example of how it will be looking a will change it right away.